flake.nix

```diff
@@ -1,24 +1,74 @@
 {
-  description = "Ghaf - documentation for TII SSRC Common Tech";
+  description = "Ghaf - documentation and implementation for TII SSRC Common Tech";
 
-  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
-  inputs.flake-utils.url = "github:numtide/flake-utils";
+  nixConfig = {
```

Comment:
The binary cache link is outdated. Please, use this one instead:
https://cache.vedenemo.dev
cache.vedenemo.dev:RGHheQnb6rXGK5v9gexJZ8iWTPX6OcSeS56YeXYzOcg=

Reply:
Done